dxr/static/templates/path_line.html

```diff
@@ -4,7 +4,7 @@
 {%- if not is_binary or is_dir -%}
 <a href="{{ url }}" {%- if is_dir %} data-path="{{ data_path }}" {% endif %}>
 {%- endif -%}
-    {{ display_path }}
+    {{ display_path|safe }}
```

Comment:
This is called from JS, right? Too bad. (Rather than putting `|safe` in the template, it's safer to return a Markup("some string") (rather than "some string") from Python.)